routes/temperature_anomalies.py

```diff
@@ -0,0 +1,171 @@
+import asyncio
+from math import floor
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+)
+
+# local imports
+from fetch_data import (
+    describe_via_wcps,
+)
+from csv_functions import create_csv
+from validate_request import (
+    get_coverage_encodings,
+    construct_latlon_bbox_from_coverage_bounds,
+    validate_latlon_in_bboxes,
+)
+from generate_urls import generate_wcs_query_url
+from generate_requests import generate_wcs_getcov_str
+from fetch_data import fetch_data, describe_via_wcps
+from postprocessing import merge_dicts, postprocess, prune_nulls_with_max_intensity
+from . import routes
+
+seaice_api = Blueprint("temperature_anomalies_api", __name__)
+
+anomaly_coverage_id = "temperature_anomaly_anomalies"
+baseline_coverage_id = "temperature_anomaly_baselines"
+
+anomaly_metadata = asyncio.run(describe_via_wcps(anomaly_coverage_id))
+baseline_metadata = asyncio.run(describe_via_wcps(baseline_coverage_id))
+
+anomaly_dim_encodings = get_coverage_encodings(anomaly_metadata)
+baseline_dim_encodings = get_coverage_encodings(baseline_metadata)
+
+
+def package_anomaly_data(point_data_list):
+    """
+    Package the temperature anomaly values into human-readable JSON format
+
+    Args:
+        point_data_list (list): nested list of data from Rasdaman WCS query
+
+    Returns:
+        di (dict): dictionary mirroring structure of nested list with keys derived from anomaly_dim_encodings global variable
+    """
+    di = dict()
+    years = list(range(1850, 2101))
```

Comment:
Observation: the year min and max could be parsed from the `anomaly_metadata` object, but I kinda like them being hard-coded for readability and because this coverage won't change in that regard.